README.md

```diff
@@ -30,3 +30,42 @@ Make company global travel and accommodation easy and convenient for the strong
 This app will be deployed on heroku, To access this app go to these link:
     - [Production URL](https://boondocks-bn-backend.herokuapp.com/)
     - [Staging](https://boondocks-bn-backend-staging.herokuapp.com/)
+
+## API Endpoints
+
+| Request Route | Methods  | Description  |
+| ------- | --- | --- |
+| /api/v1/auth/signup | POST | Users can sign up |
+| /api/v1/auth/signin | POST |  Users can sign in  |
+| /api/v1//auth/verification| GET | Users should be sent a verification email after signup |
+| /api/v1//auth/forgotPassword | POST | Users should be sent a verification email after signup |
+| /api/v1/trips/return | POST | Users can request for a return trip |
+| /api/v1/trips/oneway | POST | Users can request for a return trip |
+
+## Docker
+
+### Setting Up Docker
+
+1. Install Docker on your pc using instruction [here](https://docs.docker.com/install/). Make sure it's running well
+
+2. Clone this repo
+
+3. Follow the `.env.example` file to setup your environment and populate with corresponding values
```

Comment:
You mean `Copy`?

Reply:
Please check now I think  it's more clear